heartbeat/LVM

```diff
@@ -426,6 +426,18 @@ LVM_start() {
 	local vg=$1
 	local clvmd=0
 
+	# systemd drop-in to stop process before storage services during
+	# shutdown/reboot
+	if ps -p 1 | grep -q systemd ; then
```

Comment:
Looks like something we want a function for.

Reply:
From the man page of `sd_booted()`:

> Internally, this function checks whether the directory /run/systemd/system/ exists. A simple check like this can also be implemented trivially in shell or any other language.